intro/language/control_flow.rst

```diff
@@ -156,8 +155,10 @@ a file, ...)::
 
     >>> for i in 'powerful':
     ...     if i in vowels:
-    ...         print(i),
-    o e u
+    ...         print(i)
+    o
```

Comment:
use ``print(i, end='')`` if you wish to keep the same layout as the example before. But I think it's fine as it is now.